test/playbook.buffering.js

```diff
@@ -0,0 +1,28 @@
+// test for buffered and unbuffered output
```

Comment:
Done. I guess I was looking for a separate `functional` test suite for mocha to launch outside of the normal `npm test` suite that actually launches ansible.